src/tests/functional/plugin/shared/src/behavior/compiled_model/properties_hetero.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/compiled_model/properties_hetero.hpp"
+
+namespace ov {
+namespace test {
+namespace behavior {
+
+void OVClassHeteroCompiledModelGetMetricTest::SetCpuAffinity(ov::Core& core, std::vector<std::string>& expectedTargets) {
+#ifdef ENABLE_INTEL_CPU
+    auto layermap = core.query_model(actualNetwork, heteroDeviceName);
+    for (auto &iter : layermap) {
+        if (iter.first.find("Concat") != std::string::npos)
+            layermap[iter.first] = CommonTestUtils::DEVICE_CPU;
```

Comment:
why is hardcode for cpu here?

Reply:
I do not know exactly, but it is looks like that it does'not  metter, which device will be here, it should be just different from target. And now we use this test only for template, so CPU is applicable. But if I understand it right, if we add this test for CPU it will be not good. Let's discus it offline.